problems/variables.js

```diff
@@ -2,47 +2,85 @@
 
 // Age Calculator:
 //  * Store your birth year in a constant variable.
+const myBirthday = 1984
 //  * Store a future year in a variable.
+let futureYear = 2021
 //  * Calculate your 2 possible ages for that year based on the stored values.
 //  * For example, if you were born in 1988, then in 2026 you'll be either 37 or 38, depending on what month it is in 2026.
 //  * Log them to the screen like so: "I will be either `ageMin` or `ageMax` in `futureYear`", substituting the values.
+  let ageMin = (futureYear - myBirthday)
+  let ageMax = ((futureYear - myBirthday) + 1)
 
+  console.log("I will be either " + ageMin +" or " + ageMax)
+  
 // ## Problem Two
 
 // Snack Supply Calculator:
 //  * Store your current age in a variable.
+let currentAge = 36
 //  * Store a maximum age in a constant variable.
+const maxAge = 100
 //  * Store an estimated snack amount per day (as a number).
+let snacksPerDay = 3
 //  * Calculate how many snacks you would eat total, from your current age until the maximum age.
+console.log(((maxAge - currentAge)* 365) * snacksPerDay)
 //  * Log the result to the screen like this: "You will need `snackNumber` to last you until the age of `maxAge`".
+console.log(("You will need " + ((maxAge - currentAge)* 365) * snacksPerDay) + " to last you until the age of " + maxAge)
 
 // ## Problem Three
 
 // Calculate properties of a circle, using the definitions: http://math2.org/math/geometry/circles.htm 
 //  * Store a radius into a variable.
+let radius = 2
 //  * Calculate the circumference based on the radius, and log "The circumference is `circumferenceResult`".
+circumferenceResult = ((2 * 3.141592) * radius)
+console.log("The circumference is " + circumferenceResult)
 //  * Calculate the area based on the radius, and log "The area is `areaOfCircle`".
```

Comment:
Don't see the area. I'll mark it done but make sure you get it done

Reply:
// Calculate properties of a circle, using the definitions: http://math2.org/math/geometry/circles.htm 
//  * Store a radius into a variable.
let radius = 2
//  * Calculate the circumference based on the radius, and log "The circumference is `circumferenceResult`".
circumferenceResult = ((2 * 3.141592) * radius)
console.log("The circumference is " + circumferenceResult)
//  * Calculate the area based on the radius, and log "The area is `areaOfCircle`".
let area = (3.141592)*(radius)^2
console.log("The area of this circle is " + area)